nautobot/extras/tables.py

```diff
@@ -52,9 +52,12 @@
 )
 from .registry import registry
 
+CONTACT_OR_TEAM_ICON = """
+<i class="mdi {% if record.contact %}mdi-account{% else %}mdi-account-group{% endif %}"></i>
```

Comment:
Might make sense to add a `title="Contact"` or `title="Team"` to this element appropriately as well for mouseovers and understandability - what do you think?